source/_integrations/velux.markdown

```diff
@@ -42,6 +42,30 @@ password:
   type: string
 {% endconfiguration %}
 
+## Services
+
+### Service `velux.reboot_gateway`
+
+Reboots the configured KLF 200 Gateway.
+
+There is a problem with the KLF 200 gateway where the connection cannot be established after a restart of Home Assistant, only a manual power off and on fixes this.
+As a workaround, you can use an automation to force a restart of the KLF 200 before exiting Home Assistant, like this:
+
+```yaml
+automation:
+  alias: KLF reboot on hass stop event
+  description: ''
+  trigger:
+    - event_data: {}
+      event_type: homeassistant_stop
+      platform: event
```

Comment:
```suggestion
    - platform: homeassistant
      event: stop
```

Reply:
Just checked, I suppose the correct event is "shutdown"